releasenotes/notes/ground_state_interface-42576cb6658a46e0.yaml

```diff
@@ -18,4 +18,4 @@ deprecations:
     ``Chemistry Operator`` class is deprecated in facor of the ``tranformations``
     ``minimum_eigen_solvers/vqe_adapt`` is also deprecate and moved as an implementation
```

Comment:
```suggestion
    ``minimum_eigen_solvers/vqe_adapt`` is also deprecated and moved as an implementation
```

Reply:
I don't think this change belongs here. Let's rather at that in #1357

Reply:
ok

Reply:
I was actually 10 seconds too slow with pushing the fix on that branch. There is also more typos in the reno. @woodsp-ibm do you want to include them here instead? It is very out of place but it feels like an unnecessary change for another PR

Reply:
Now I notice favor is mispelled too - lets do it here since its in this set of changed files

Reply:
I will push the commit I made earlier but couldn't push anymore.

As a sidenote: we are working on the reno as we speak. Should be there within the next minutes

Reply:
Sorry I was on the wrong branch. It is now in the excited states PR
And that's also where the reno is coming soon